declarations.d.ts

```diff
@@ -0,0 +1,16 @@
+interface Window {
```

Comment:
This code mostly seems to favor `type` over `interface`; why `interface` here?

Reply:
Good q. Sticking to `type` seems worthwile